chromium_src/chrome/browser/ui/webui/whats_new/whats_new_fetcher.cc

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "chrome/browser/ui/webui/whats_new/whats_new_fetcher.h"
+
+#include "brave/browser/ui/whats_new/whats_new_util.h"
+
+#define StartWhatsNewFetch StartWhatsNewFetch_UnUsed
```

Comment:
same as the other overrides we don't need to include the original file when we don't care about the original functionality at all. We should replace `StartWhatsNewFetch` with the contents of `StartBraveWhatsNew` and also override `GetServerURL` to be the correct url for Brave. I pinged @simonhong though because I'm confused about why we're opening a new tab directly instead of fetching like chromium